src/main/java/com/matera/account/accounttype/AccountType.java

```diff
@@ -0,0 +1,14 @@
+package com.matera.account.accounttype;
+
+import javax.persistence.*;
+
+@Entity
+public class AccountType {
+
+    @Id
+    public int accountTypeId;
+    public String accountType;
+
+}
+
+
```

Comment:
It might be useful to have Getter's & Setter's to accountTypeId and accountType.